api/machine/machine_driver.go

```diff
@@ -44,22 +48,78 @@ func (h *Handlers) ActionHandler(actionName string, action *types.Action, apiCon
 	return nil
 }
 
-// Formatter for MachineDriver
-func (h *Handlers) Formatter(apiContext *types.APIContext, resource *types.RawResource) {
+// FormatterDriver for MachineDriver
+func (h *Handlers) FormatterDriver(apiContext *types.APIContext, resource *types.RawResource) {
```

Comment:
@ibuildthecloud on management-api, for related types like machine, machineConfig, machineDriver - would you prefer to have folder/file per file (this way we can maintain standard names like Validate/Formatter, etc). Or keep them in the same folder/file, and reflect type in method name